src/components/Link.tsx

```diff
@@ -1,228 +1,126 @@
 import React from "react"
-import { Link as GatsbyLink, navigate as gatsbyNavigate } from "gatsby"
+import { Icon, Link, LinkProps, useTheme } from "@chakra-ui/react"
+import { navigate as gatsbyNavigate } from "gatsby"
 import { LocalizedLink as IntlLink } from "gatsby-theme-i18n"
 import { NavigateOptions } from "@reach/router"
 import { IntlShape } from "react-intl"
-import styled from "@emotion/styled"
 
-import Icon from "./Icon"
+import { BsQuestionSquareFill } from "react-icons/bs"
 
-import { isLang, Lang } from "../utils/languages"
+import { Lang } from "../utils/languages"
 import { trackCustomEvent, EventOptions } from "../utils/matomo"
 import { Direction } from "../types"
 
 const HASH_PATTERN = /^#.*/
-// const DOMAIN_PATTERN = /^(?:https?:)?[/]{2,}([^/]+)/
-// const INTERNAL_PATTERN = /^\/(?!\/)/
-// const FILE_PATTERN = /.*[/](.+\.[^/]+?)([/].*?)?([#?].*)?$/
 
 const isHashLink = (to: string): boolean => HASH_PATTERN.test(to)
 
-const ExternalLink = styled.a`
-  &:after {
-    margin-left: 0.125em;
-    margin-right: 0.3em;
-    display: inline;
-    content: "↗";
-    transition: all 0.1s ease-in-out;
-    font-style: normal;
-  }
-  &:hover {
-    &:after {
-      transform: translate(0.15em, -0.2em);
-    }
-  }
-`
-
-const InternalLink = styled(IntlLink)`
-  .is-glossary {
-    white-space: nowrap;
-  }
-  &.active {
-    color: ${(props) => props.theme.colors.primary};
-  }
-  &:hover {
-    svg {
-      fill: ${(props) => props.theme.colors.primary};
-      transition: transform 0.1s;
-      transform: scale(1.2);
-    }
-  }
-`
-
-const ExplicitLangInternalLink = styled(GatsbyLink)`
-  &.active {
-    color: ${(props) => props.theme.colors.primary};
-  }
-`
-
-const GlossaryIcon = styled(Icon)`
-  margin: 0 0.25rem 0 0.35rem;
-  fill: ${(props) => props.theme.colors.primary400};
-  text-decoration: underline;
-  &:hover {
-    transition: transform 0.1s;
-    transform: scale(1.2);
-  }
-`
-
-export interface IProps {
+export interface IBaseProps {
   to?: string
   href?: string
-  dir?: Direction
   hideArrow?: boolean
-  className?: string
   isPartiallyActive?: boolean
-  ariaLabel?: string
   customEventOptions?: EventOptions
-  onClick?: () => void
 }
 
-const Link: React.FC<IProps> = ({
-  to,
-  dir = "ltr",
+export interface IProps extends IBaseProps, LinkProps {
+  dir?: Direction // TODO: remove this prop once we use the native Chakra RTL support
+}
+
+const LinkWrapper: React.FC<IProps> = ({
+  to: toProp,
   href,
+  dir = "ltr",
   children,
   hideArrow = false,
-  className,
   isPartiallyActive = true,
-  ariaLabel,
   customEventOptions,
-  onClick = () => {},
+  ...restProps
 }) => {
-  // markdown pages pass `href`, not `to`
-  to = to || href
+  const theme = useTheme()
 
-  if (!to) {
-    throw new Error("Either 'to' or 'href' props must be provided")
-  }
+  // TODO: in the next PR we are going to deprecate `to` prop and just use `href`
+  const to = (toProp || href)!
```

Comment:
in a following PR we are going to just use `href` for everything

Reply:
SGTM